internal/command/container.go

```diff
@@ -0,0 +1,398 @@
+package command
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/fyne-io/fyne-cross/internal/icon"
+	"github.com/fyne-io/fyne-cross/internal/log"
+	"github.com/fyne-io/fyne-cross/internal/volume"
+	"golang.org/x/sys/execabs"
+)
+
+const (
+	// fyneBin is the path of the fyne binary into the docker image
+	fyneBin = "/usr/local/bin/fyne"
+	// gowindresBin is the path of the gowindres binary into the docker image
+	gowindresBin = "/usr/local/bin/gowindres"
+)
+
+type Debugger interface {
+	Debug(v ...interface{})
+	Debugging() bool
+}
+
+type ContainerEngine interface {
+	CreateContainerImage(arch Architecture, OS string, image string) ContainerImage
```

Comment:
It looks like `CreateContainerImage` could be moved as package function having it to accept a `ContainerEngine` argument. In this way we could remove it from the interface. Something like:
```CreateContainerImage(ce ContainerEngine, arch Architecture, OS string, image string) ContainerImage```

Reply:
I tried to figure how to make this work, but CreateContainerImage is allowing each engine to customize its behavior for a ContainerImage. I don't see how to do the dispatching properly without having it in the interface or at least a version of it in the interface, do you have any suggestion?

Reply:
The idea was to have it as a package function.
Could that work?

Reply:
Trying to think about it, but the idea would be to move the function and the engine into a package for that specific engine? It would add an additional dispatching function that every OS specific file would use to route the call between different engine. Seems not ideal to me, but doable. Do I understand what you have in mind correctly?